Eastern-Research-Group/mywaterway
# feature/community-widget-relative-paths

## Related Issues:
* https://jira.epa.gov/browse/HMW-3

## Main Changes:
* Updated the community-widgets.html file to use relative paths 
* Updated the example paths to what will be on production

## Steps To Test:
1. Navigate to http://localhost:9090/widgets/community-widgets.html
2. Verify the widgets still load
